assopy/models.py

```diff
@@ -181,6 +183,9 @@ def wrapper(instance, filename):
 ticket_for_user = dispatch.Signal(providing_args=['tickets'])
 
 class User(models.Model):
+    """
+    aka. AssopyUser
```

Comment:
what does this mean? :)

Reply:
there are multiple models called User (specifically this one, and the django builtin), and I noticed that there is a factory for this model that's often imported as AssopyUserFactory; so I "tagged" this one which makes it easier to find it when needed. I'd ideally rename it (or delete it, and replace this, and AttendeeProfile with single custom user model) but that's beyond the scope of this PR. ;)

Reply:
also, it's named User but it's a UserProfile (using django terminology, from pre 1.5) not a standalone User model.

Reply:
maybe we should add this explanation to the source file :)

Reply:
that's a great idea. :) I'll put in there later today.